hollow/src/main/java/com/netflix/hollow/api/codegen/HollowAPIGenerator.java

```diff
@@ -334,6 +345,15 @@ public Builder reservePrimaryKeyIndexForTypeWithPrimaryKey(boolean reservePrimar
             return this;
         }
 
+        /**
```

Comment:
Wonder if it would be good idea to internally invoke withErgonimicsShortcusts and reflect that in javadoc